lib/kapost-provision/commands/cabbage.rb

```diff
@@ -86,7 +97,9 @@ def create_heroku_default_config(heroku_name, config)
 
   def shell(command)
     unless system(command)
-      error("The last command failed with error status #{$?}")
+      msg = "The last command failed with error status #{$?}"
+      # Raise, so its rescuable
+      raise Heroku::Command::CommandFailed.new(msg)
```

Comment:
Honest question: overall, is it appropriate to raise an error defined externally?